source/material_model/interface.cc

```diff
@@ -747,6 +747,46 @@ namespace aspect
       return vs;
     }
 
+    namespace
+    {
+      std::vector<std::string> make_plastic_additional_outputs_names()
+      {
+        std::vector<std::string> names;
+        names.push_back("weakened_cohesions");
+        names.push_back("weakened_friction_angles");
```

Comment:
I am not quite happy with the term `weakened`, what if people implement a healing algorithm that might make the values higher than the initial values? Would you be ok with `current_...`?